AppCenter/AppCenter/MSServiceAbstract.m

```diff
@@ -45,7 +46,7 @@ - (void)setEnabled:(BOOL)isEnabled {
 - (void)applyEnabledState:(BOOL)isEnabled {
 
   // Propagate isEnabled and delete logs on disabled.
-  [self.logManager setEnabled:isEnabled andDeleteDataOnDisabled:YES forGroupId:self.groupId];
+  [self.channelUnit setEnabled:isEnabled andDeleteDataOnDisabled:YES];
```

Comment:
I really like how now each module references it's own channel instead of passing around the `groupID`.